src/app/pages/device-options/blind-percentage/blind-percentage.component.ts

```diff
@@ -0,0 +1,128 @@
+import {
```

Comment:
ce composant ressemble beaucoup au dim level, il était pas réutilisable ?

Reply:
Yeah, this component is very similar with the dim level. I believe there is a better method which can share it with dim level. I will figure it out.